pymetis/__init__.py

```diff
@@ -257,20 +263,47 @@ def part_mesh(n_parts, connectivity, options=None):
 
     n_elements = len(connectivity)
     n_vertex = len(set(conn))
-
+    
+   
     # Handle option validation
     if options is None:
         options = Options()
+        #set default values
```

Comment:
Why is this a good idea? Metis has a routine to set the defaults. Why do we have a diverging opinion?

Reply:
METIS has CLI tools to partition graph/mesh (`gpmetis` and `mpmetis`). These commands use some different default option values. Previously, I assumed that these values are more friendly to new METIS users, compared with `METIS_SetDefaultOptions`, which simply assigns -1 to everything.

I am not insisting on this part. I will remove them if you think it brings more trouble than benefits.

Reply:
If this is a functional change, then this is even worse; it might change the behavior of people's code behind their back. We probably shouldn't do this.